src/client/pages/WelcomeSocial.tsx

```diff
@@ -64,19 +68,24 @@ const inlineMessage = (socialProvider: SocialProvider) => css`
 
 export type WelcomeSocialProps = RegistrationProps & {
 	socialProvider: SocialProvider;
-	isNativeApp?: IsNativeApp;
+	geolocation?: GeoLocation;
 };
 
 export const WelcomeSocial = ({
 	queryParams,
 	formError,
 	socialProvider,
-	isNativeApp,
+	geolocation,
 }: WelcomeSocialProps) => {
 	const formTrackingName = 'register';
 
 	usePageLoadOphanInteraction(formTrackingName);
 
+	// don't show the Saturday Edition newsletter option for US and AU
+	const showSaturdayEdition = !(['US', 'AU'] as GeoLocation[]).some(
+		(location: GeoLocation) => location === geolocation,
+	);
```

Comment:
Is it just US and AU that are excluded or everywhere except GB?

Reply:
Just US and AU